lib/modules/manager/circleci/schema.ts

```diff
@@ -9,9 +9,33 @@ export const CircleCiJob = z.object({
   docker: z.array(CircleCiDocker).optional(),
 });
 
+type Orb = {
+  executors?: Record<string, CircleCiJob>;
+  jobs?: Record<string, CircleCiJob>;
+  orbs?: Record<string, string | Orb>;
+};
+
+type CircleCiJobInput = z.input<typeof CircleCiJob>;
```

Comment:
```suggestion
type CircleCiJob = z.input<typeof CircleCiJob>;
```

don't rename and move to const (below).

https://github.com/renovatebot/renovate/blob/main/docs/development/zod.md#inferred-types

Reply:
I think in this specific case it actually isn't needed to separate out input/output types, removed it